Ewl 3574 ec template

chelsiejohnston opened the pull request:
## Ticket(s)

**Github Issue**

N/A 

**Jira Ticket**

- [Issue 3574: EWL-3574 EC | Create Education Center Marketing Page Template](https://issues.ama-assn.org/browse/EWL-3574)


## Description

Pulls child patterns into and cleans up the EC Home Template pattern:
- Establishes a `ec-home.json` file for the pattern whose content matches the wireframes
- Makes it so that the included patterns can either inherit content data by way of being included in `ec-home.twig` and provides fallbacks to default dummy content if not
- Adds preliminary styling for the EC Home Template and its child patterns when included in the template
- Adds .json files to some other larger patterns (hub things mostly)
- Adds conditionals to the `ec-card-row` layout patterns so they can show different card types depending on what's passed to them


## To Test

- [ ] Put on your testin' boots
- [ ] Go to Templates > EC Home > EC Marketing Home Page (Three-Up) and verify that it responds appropriately at all breakpoints, and in correlation with the [wireframes](http://lleoij.axshare.com/) 
- [ ] Go to Templates > EC Home > EC Marketing Home Page (Four-Up) and verify that it responds appropriately at all breakpoints, and in correlation with the [wireframes](http://lleoij.axshare.com/) 
- [ ] Go to Templates > EC Home > EC Marketing Home Page (Five-Up) and verify that it responds appropriately at all breakpoints, and in correlation with the [wireframes](http://lleoij.axshare.com/) 
- [ ] Look at some of the smaller patterns included in this template and make sure they show appropriate dummy content when not included in the template: for example, Molecules > Sections > EC Card Row > three-up, four-up, and five-up-left patterns
- [ ] See that the .md information (Pattern Info) and annotations for these templates make sense

## Relevant Screenshots/GIFs

![3574-1](https://user-images.githubusercontent.com/12160398/27646702-ec11b8e8-5bee-11e7-8f0c-069655bb7317.gif)



## Remaining Tasks

- [x] In order to work on this I needed to pull in the code from https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/151 - I'll update when that's wrapped up, or someone else can later if we want to merge this in before that.
- [x] Replace ipsumimage
- [x]  (?) The Course Browser pattern has an `h1` for its heading element. This should probably be an `h2` at the very least. (Check with Megh)
- [x] Standardize wrapper markup between sections - right now, most things are `sections`, but some aren't. There are also differences in where the `layout` class is applied, if at all.
- [x]  (?) The hero shown in the wireframes has an overlay, but the no-overlay one is used here. Did I just screw this up on my own? Looking into it.
- [x] Image landscape cards need a fixed height.
- [x] how to show other variants of layouts? (three-up, five-up, lefts?) - split them out; add general annotations to the pattern, pointing out the sub-patterns and variants. Create three variants (three-up, four-up, five-up) of template 
- [x] Uses placeholder text from Wireframes. Add some .md text. 
- [ ] ~x-out search button in weird spot (probably just leave this until Andy reviews)~ (wait until corp review to see if this is really an issue)
- [ ] Put Courses section into .json (optional, but nice to have) 


## Additional Notes

- LOE for this should've been 16.

chelsiejohnston commented:
I've wrapped up the things that were outstanding, as well as added details to the PR, so this is officially ready for review.

rpama commented:
**Desktop/General**
 - [ ] Hero: button needs to be contained within transparent overlay
 - [ ] Hero: text inside placeholder image needs to be smaller to prevent cutoff

![hero_compare](https://user-images.githubusercontent.com/28711067/27657041-e9f4b502-5c10-11e7-8deb-5b984817bfc0.jpg)

 - [ ] Alert: this section is missing
 - [ ] "Accessible" section: Remove devices image and use placeholder (this is a template, not a page)
 - [ ] "Accessible" section: content placement in columns is reverse of wireframes
 - [ ] "Accessible" section: content is spaced rather more generously than in wireframes

![accessible_compare](https://user-images.githubusercontent.com/28711067/27657519-8ac92a0c-5c12-11e7-933b-2877e1200b55.png)

 - [ ] Footer: "Mission" text font is serif but wireframe shows sans-serif (not sure if this was a deliberate design choice in the wireframes or a mistake)

rpama commented:
**Portrait**
 - [ ] Card-ctas: spacing/card height isn't preserved quite the same as in wireframes

![card-cta_comparison](https://user-images.githubusercontent.com/28711067/27658622-49b8f96c-5c16-11e7-9202-0219c9b36235.jpg)

 - [ ] Course cards: Text alignment doesn't match up with wireframes

![course-card_comparison](https://user-images.githubusercontent.com/28711067/27658829-fb3902cc-5c16-11e7-8b2a-d9460b827b84.jpg)

- [ ] Course Bundle cards: Layout is not correct

![course-bundles_comparison](https://user-images.githubusercontent.com/28711067/27659045-b3a0b828-5c17-11e7-9714-155a3bad7033.jpg)

 - [ ] "Accessible" section: image seems too large, maybe give a max width of ~80%

rpama commented:
**Mobile**
 - [ ] Hero: "start a course" button is bottom aligned in the wireframe
 - [ ] Hero: should extend fully the width of the screen

![hero_mobile_comparison](https://user-images.githubusercontent.com/28711067/27659511-610e2cb0-5c19-11e7-9dd2-551465743a34.jpg)

 - [ ] Card-ctas: spacing/card height isn't preserved quite the same as in wireframes (see Portrait notes)
 - [ ] Course cards: text should be left aligned, not centered (except duration)
 - [ ] Course bundle cards: title should be left aligned, not centered
 - [ ] Course bundle cards: should extend fully the width of the screen
 - [ ] Course bundle cards/Testimonial: there should not be a dividing line between these
 - [ ] Testimonial: wireframes show a custom counter on mobile, is that needed?
 
![screen shot 2017-06-28 at 3 53 49 pm](https://user-images.githubusercontent.com/28711067/27659955-e7dfb384-5c1a-11e7-9375-fde28de47cee.png)

 - [ ] Testimonial/ "Accessible" section: there should not be a dividing line between these

chelsiejohnston commented:
@rpama thanks - I bolded the items in your above comments that I feel good about addressing immediately. The rest I've responded to below: 

> Hero: button needs to be contained within transparent overlay
> ![hero_compare](https://user-images.githubusercontent.com/28711067/27657041-e9f4b502-5c10-11e7-8deb-5b984817bfc0.jpg)

I included an existing hero component here because that's what the wireframes specified in an annotation. To avoid creating extra work/an extra pattern variant while trying to get this PR merged, I'd like to wait on Andy for this to see if this change is totally necessary.

> "Accessible" section: content is spaced rather more generously than in wireframes
> ![accessible_compare](https://user-images.githubusercontent.com/28711067/27657519-8ac92a0c-5c12-11e7-933b-2877e1200b55.png)

I based this spacing off of the spacing from the VSA comps and some of the hub pages, which the wireframes do not match. Also, some of these padding/margins got messed up due to existing styling in the smaller patterns that I didn't have time to alter. Before doing that work I'd like to wait on Andy for this so we can get some definition around what the spacing should actually be for this section.

> Footer: "Mission" text font is serif but wireframe shows sans-serif (not sure if this was a deliberate design choice in the wireframes or a mistake)

Let's wait on Andy for this. I'm not sure this was an intentional declaration in the wireframes.

> "Accessible" section: image seems too large, maybe give a max width of ~80%

Would like Andy to weigh on a design decision for this one.

> Hero: "start a course" button is bottom aligned in the wireframe
> ![hero_mobile_comparison](https://user-images.githubusercontent.com/28711067/27659511-610e2cb0-5c19-11e7-9dd2-551465743a34.jpg)

Same comment here as my comment in the Desktop section about the hero.

> Testimonial: wireframes show a custom counter on mobile, is that needed?
> ![screen shot 2017-06-28 at 3 53 49 pm](https://user-images.githubusercontent.com/28711067/27659955-e7dfb384-5c1a-11e7-9375-fde28de47cee.png)

I have no idea! Let's check with Andy on Friday...

chelsiejohnston commented:
> "Accessible" section: content placement in columns is reverse of wireframes 

There are two variants, a left and right variant. I wasn't sure how to show this in the pattern, but it's described in the .md?

rpama commented:
**4 up layout, desktop**

I wonder if we need to lower the font size or lower the character limits for titles since I'm not sure we're getting the desired effect at smaller size ranges.

![screen shot 2017-06-29 at 10 16 05 am](https://user-images.githubusercontent.com/28711067/27695365-4385204a-5cb4-11e7-889c-274840099454.png)

chelsiejohnston commented:
> Card-ctas: spacing/card height isn't preserved quite the same as in wireframes
After a little work, I think we should wait for Andy on this one. This is a problem with cards that have text, and it may be more of a design problem. I think that if we show him the actual behavior of the once-static cards now that they're in a browser, he can help make decisions around card heights/text.

![3574-6](https://user-images.githubusercontent.com/12160398/27697066-dbcb06f4-5cb8-11e7-8caa-4992f3192f56.gif)

rpama commented on styleguide/source/_patterns/01-molecules/06-components/20-card-image-landscape/_card-image-landscape.scss:
```diff
@@ -1,98 +1,53 @@
-// Styling for image landscape cards when they're included in rows.
+.card-image-landscape_description {
 
-.layout_three_up-row > *,
```
Since we have separate row layouts for 3, 4, and 5 up and these are landscape and don't have the same height issues as some other cards, I don't think we actually need these styles which is why I removed them.

rpama commented on styleguide/source/_patterns/02-organisms/05-ec/01-ec-search-bar/_ec-search-bar.scss:
```diff
@@ -1,42 +1,43 @@
-.ec-search-bar {
+.ec_search-bar {
```
I disagree with this renaming. I believe the block's complete name should be `ec-search-bar` as it does not make sense to treat these as separable into components.

chelsiejohnston replied:
Leaving this as-is for now

rpama commented on styleguide/source/_patterns/02-organisms/05-ec/01-ec-search-bar/ec-search-bar.twig:
```diff
@@ -1,10 +1,8 @@
-    <div class="ec-search-bar_form">
-      {% include "molecules-search-field" with {'placeholder': 'Search Courses'} %}
-    </div>
+<div class="ec_search-bar_inner layout">
```
I'm not sure I understand the refactoring here. It's one thing to refactor the pattern to use `<sections>` rather than `<div>` but quite another to fragment the pattern, which is what has happened here. Please return the parent container with the class `ec_search-bar` to the pattern.

chelsiejohnston replied:
[commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/5b1b699b0424b74ca50c3a9381957555b55a2cfb)

rpama commented on styleguide/source/_patterns/03-templates/02-ec-home/01-ec-home-three-up/ec-home-three-up.twig:
```diff
@@ -0,0 +1,50 @@
+{% include 'molecules-alert-info' %}
+
+<div class="ec">
+    <section class="ec_search-bar">
```
As noted above, `<section class="ec_search-bar">` should be part of the ec-search-bar pattern.

chelsiejohnston replied:
[commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/5b1b699b0424b74ca50c3a9381957555b55a2cfb)

rpama commented on styleguide/source/_patterns/03-templates/02-ec-home/01-ec-home-three-up/ec-home-three-up.twig:
```diff
@@ -0,0 +1,50 @@
+    <section class="ec_cards-landscape">
+        {% include 'organisms-card-row-ec-three-up' with {'cardType': 'landscape'} %}
+        {#{% include 'organisms-card-row-ec-four-up' with {'cardType': 'landscape'} %}#}
+        {#{% include 'organisms-card-row-ec-five-up-left' with {'cardType': 'landscape'} %}#}
```
Please remove commented out lines of code and refactor `with {'cardType': 'landscape'}` according to notes in the card-row files above.

rpama commented on styleguide/source/_patterns/03-templates/02-ec-home/01-ec-home-three-up/ec-home-three-up.twig:
```diff
@@ -0,0 +1,50 @@
+    </section>
+
+    <section class="ec_banner-cta-no-stacking layout">
+        {#{% include 'molecules-banner-cta-no-stacking' with { 'content': ctaBanner } %}#}
```
remove commented code

rpama commented on styleguide/source/_patterns/03-templates/02-ec-home/01-ec-home-three-up/ec-home-three-up.twig:
```diff
@@ -0,0 +1,50 @@
+    </section>
+
+    <section class="ec_testimonial layout">
+        {#{% include 'molecules-testimonial' with { 'content': testimonial } %}#}
```
remove commented code

rpama commented on styleguide/source/_patterns/03-templates/02-ec-home/02-ec-home-four-up/ec-home-four-up.twig:
```diff
@@ -0,0 +1,50 @@
+    <section class="ec_cards-landscape">
+        {#{% include 'organisms-card-row-ec-three-up' with {'cardType': 'landscape'} %}#}
+        {% include 'organisms-card-row-ec-four-up' with {'cardType': 'landscape'} %}
+        {#{% include 'organisms-card-row-ec-five-up-left' with {'cardType': 'landscape'} %}#}
```
remove commented code, refactor based on recommended card-row-edits

rpama commented on styleguide/source/_patterns/03-templates/02-ec-home/02-ec-home-four-up/ec-home-four-up.twig:
```diff
@@ -0,0 +1,50 @@
+
+    <section class="ec_testimonial layout">
+        {% include 'molecules-testimonial' with { 'content': testimonial } %}
+        {#{% include 'molecules-testimonial-left' with { 'content': testimonial } %}#}
```
remove commented code

rpama commented on styleguide/source/_patterns/03-templates/02-ec-home/02-ec-home-four-up/ec-home-four-up.twig:
```diff
@@ -0,0 +1,50 @@
+
+    <section class="ec_banner-cta-no-stacking layout">
+        {% include 'molecules-banner-cta-no-stacking' with { 'content': ctaBanner } %}
+        {#{% include 'molecules-banner-cta-no-stacking-left' with { 'content': ctaBanner } %}#}
```
remove commented code

rpama commented on styleguide/source/_patterns/03-templates/02-ec-home/03-ec-home-five-up/ec-home-five-up.twig:
```diff
@@ -0,0 +1,50 @@
+    <section class="ec_cards-landscape">
+        {#{% include 'organisms-card-row-ec-three-up' with {'cardType': 'landscape'} %}#}
+        {#{% include 'organisms-card-row-ec-four-up' with {'cardType': 'landscape'} %}#}
+        {% include 'organisms-card-row-ec-five-up-left' with {'cardType': 'landscape'} %}
```
remove commented code, refactor

rpama commented on styleguide/source/_patterns/03-templates/02-ec-home/03-ec-home-five-up/ec-home-five-up.twig:
```diff
@@ -0,0 +1,50 @@
+
+    <section class="ec_testimonial layout">
+        {% include 'molecules-testimonial' with { 'content': testimonial } %}
+        {#{% include 'molecules-testimonial-left' with { 'content': testimonial } %}#}
```
remove commented code

rpama commented on styleguide/source/_patterns/03-templates/02-ec-home/03-ec-home-five-up/ec-home-five-up.twig:
```diff
@@ -0,0 +1,50 @@
+
+    <section class="ec_banner-cta-no-stacking layout">
+        {% include 'molecules-banner-cta-no-stacking' with { 'content': ctaBanner } %}
+        {#{% include 'molecules-banner-cta-no-stacking-left' with { 'content': ctaBanner } %}#}
```
remove commented code

chelsiejohnston commented on styleguide/source/_patterns/01-molecules/06-components/11-banner-cta/banner-cta-no-stacking.twig:
```diff
@@ -1,8 +1,8 @@
-<article class="banner-cta-no-stacking">
-    {% include 'atoms-h1' with { 'content': 'Lorem ipsum dolor sit amet', 'class': 'banner-cta-no-stacking_title' } %}
-    <div class="banner-cta-no-stacking_image" style="background-image: url(../../assets/images/fpo_3x2.png)"></div>
+<div class="banner-cta-no-stacking">
```
Addressed the ipsumimage in [https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/f0440da79227576ef8cef72ff5dfbde7123240d1](this commit.)

chelsiejohnston replied:
[Addressed the conditionals in this commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/842895a15dfaece80a375805b274c3690aa58c0c)

chelsiejohnston commented on styleguide/source/_patterns/01-molecules/06-components/20-testimonial/testimonial-left.twig:
```diff
@@ -3,10 +3,10 @@
       <p class="testimonial_speaker">
-          <span class="testimonial_speaker_person">—Dr. Bob Jones</span>,
-          <span class="testimonial_speaker_title">Medicology</span>
+          <span class="testimonial_speaker_person">{{ testimonial.speakerName ? testimonial.speakerName : 'Bob M. Doctor' }}</span>,
```
I agree and addressed this in [this commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/45352a2216715cf97527f1e7414440a8b93e9d2d)

chelsiejohnston commented on styleguide/source/_patterns/01-molecules/06-components/20-testimonial/testimonial.twig:
```diff
@@ -1,12 +1,12 @@
       <p class="testimonial_speaker">
-          <span class="testimonial_speaker_person">—Dr. Bob Jones</span>,
-          <span class="testimonial_speaker_title">Medicology</span>
+          <span class="testimonial_speaker_person">{{ testimonial.speakerName ? testimonial.speakerName : 'Bob M. Doctor' }}</span>,
```
addressed this in [this commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/45352a2216715cf97527f1e7414440a8b93e9d2d)

chelsiejohnston commented on styleguide/source/_patterns/02-organisms/03-sections/02-card-row-ec/01-card-row-ec-three-up.twig:
```diff
@@ -1,13 +1,26 @@
-    <div class="layout_item layout_three_up-primary">
-      {% include 'molecules-card-image-landscape' %}
+    <div class="layout_three_up-row">
+        {% if cardType == 'landscape' %}
```
Addressed in [this commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/42856000e883b34e26e6d70712e55897e751ef76).

chelsiejohnston commented on styleguide/source/_patterns/02-organisms/03-sections/02-card-row-ec/02-card-row-ec-four-up.twig:
```diff
@@ -1,6 +1,30 @@
-    <div class="layout_item">{% include 'molecules-card-image-landscape' %}</div>
-</div>
\ No newline at end of file
+{% if cardType == 'landscape' %}
```
I like that this is more succinct, but it might not be immediately obvious that landscape cards are the default here. This is probably fine and someone familiar with Twig syntax should be be able to figure out what's going on with some poking around.

Wanted to point that out; despite this, I've made these changes [here](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/42856000e883b34e26e6d70712e55897e751ef76).

chelsiejohnston commented on styleguide/source/_patterns/02-organisms/03-sections/02-card-row-ec/03-card-row-ec-five-up-left.twig:
```diff
@@ -1,9 +1,19 @@
 <div class="layout layout_five-up-left">
-  <div class="layout_item">{% include 'molecules-card-image-landscape' %}</div>
+  {% if cardType == 'landscape' %}
+  <div class="layout_item">{% include 'molecules-card-image-landscape' with {'cardRowContent': cardRowContent.cardsLandscape.first} %}</div>
```
Addressed in [this commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/42856000e883b34e26e6d70712e55897e751ef76).

chelsiejohnston commented on styleguide/source/_patterns/00-atoms/04-images/01-landscape-3x2.twig:
```diff
@@ -1 +1 @@
-<img {% if class %}class='{{ class }}'{% endif %} src="{{ src|default('../../assets/images/fpo_3x2.png') }}" alt="This is an image.">
```
[commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/72a8d0aec907b6be7425f1da7b0914287c64bbc1)

chelsiejohnston commented on styleguide/source/_patterns/01-molecules/06-components/10-card-cta/card-cta-no-image.twig:
```diff
@@ -1,4 +1,4 @@
 <a class="card-cta" href="#">
-  {% include 'atoms-h2' with { 'class': 'card-cta_title', 'content' : 'Leadership & Elections' } %}
-  {% include 'atoms-paragraph' with { 'class': 'card-cta_desc', 'content' : 'View the results of the election—the new governing council is ready to serve the Women Physician Section. View the results of the election—the new governing council is ready to serve the Women Physician Section.' } %}
+    {% include 'atoms-h2' with { 'class': 'card-cta_title card-cta_title-no-border', 'content': cardRowContent.title ? cardRowContent.title : 'Leadership & Elections'} %}
```
[commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/842895a15dfaece80a375805b274c3690aa58c0c)

chelsiejohnston commented on styleguide/source/_patterns/01-molecules/06-components/11-banner-cta/banner-cta-no-stacking-left.twig:
```diff
@@ -1,8 +1,8 @@
-    {% include 'atoms-h1' with { 'content': 'Lorem ipsum dolor sit amet', 'class': 'banner-cta-no-stacking_title' } %}
-    <div class="banner-cta-no-stacking_image" style="background-image: url(../../assets/images/fpo_3x2.png)"></div>
+<div class="banner-cta-no-stacking banner-cta-no-stacking-left">
+    {% include 'atoms-h2' with { 'content': ctaBanner.heading ? ctaBanner.heading : 'Lorem ipsum dolor sit amet', 'class': 'banner-cta-no-stacking_title h1' } %}
```
[commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/842895a15dfaece80a375805b274c3690aa58c0c)

chelsiejohnston commented on styleguide/source/_patterns/01-molecules/09-hub-components/00-hub-hero/hub-hero-no-overlay.twig:
```diff
@@ -1,9 +1,9 @@
   <div class="hub-hero-inner">
-    {% include 'atoms-h1' with { 'content': 'Lorem ipsum dolor sit amet', 'class': 'hub-hero_title' } %}
-    {% include 'atoms-paragraph' with { 'content': 'Gain an active voice on politics protecting patients and your profession.', 'class': 'hub-hero_desc' } %}
+    {% include 'atoms-h1' with { 'content': heroContent.heading ? heroContent.heading : 'Lorem ipsum dolor sit amet', 'class': 'hub-hero_title' } %}
```
[commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/842895a15dfaece80a375805b274c3690aa58c0c)

chelsiejohnston commented:
I've addressed all PR feedback and visual QA feedback from Rachel/Andy. Considering this done.

If someone wants to review it again to take a final look, lmk. I'm adding the "ready for review" label again.

rpama commented:
**3 up / General**
- [ ] Course cards: Text alignment doesn't match up with wireframes --> still seeing this issue & noted above in the "portrait" section (by which I'm sure I meant "tablet" :) )
- [ ] Course & course bundle cards: Mobile styles use alternating background colors per row, but the card with white background are not showing any kind of hover state.
![bundle-cards-hover](https://user-images.githubusercontent.com/28711067/27920206-7e597f56-6239-11e7-81da-a0842be6006e.gif)
- [ ] Course bundle cards/Testimonial: there should not be a dividing line between these on table and smaller but I am still seeing the divider
- [ ] Browse tabs menu - wrapping at small desktop sizes. Is this ok?
![screen shot 2017-07-06 at 11 11 17 am](https://user-images.githubusercontent.com/28711067/27920986-f897db08-623b-11e7-84a6-c219d4080e3c.png)

** 4 up layout **
- [ ] 4 up layout [card overlays comment](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157#issuecomment-311999802) is still an issue. Didn't we discuss hiding the description field on Wed?
- [ ] 4 up layout Card background images are being stretched instead of preserving the landscape aspect ratio
![screen shot 2017-07-06 at 11 26 40 am](https://user-images.githubusercontent.com/28711067/27921729-4f4c5c10-623e-11e7-99c3-9c58714e1dc8.png)

**5 up layout**
- [ ] 5 up layout cards desktop: big card touches/overlaps at some sizes (shown here at 1230px width)
![screen shot 2017-07-06 at 11 29 36 am](https://user-images.githubusercontent.com/28711067/27921799-9060e81a-623e-11e7-91ce-69bc3652d430.png)
- [ ] 5 up layout tablet: card images not keeping their aspect ratios as noted above
- [ ] 5 up layout tablet: card row edges do not align with hero
![screen shot 2017-07-06 at 11 39 04 am](https://user-images.githubusercontent.com/28711067/27922246-00b13614-6240-11e7-969c-c01981437b5f.png)
- [ ] 5 up layout tablet: does not look like the wireframes
![5uptablet](https://user-images.githubusercontent.com/28711067/27922643-4da71816-6241-11e7-9360-28e22a216d02.jpg)
- [ ] 5 up layout mobile: now all 5 cards are shown in the wireframes instead of just 3 (did this get changed?!). So SG is not matching wireframes now.

rpama commented:
Stopping the review with the visual notes above so we can get those marked as resolved. Will do a last review of the actual code once we've got everything nailed down.

chelsiejohnston commented:
@rpama back to you

rpama commented:
Visually, I think this is looking good! Will do one more pass through the code and then hopefully be merging this.
